aptos-move/aptos-vm/src/sharded_block_executor/tests.rs

```diff
@@ -7,6 +7,9 @@ use crate::{
     },
     ShardedBlockExecutor,
 };
+use aptos_block_partitioner::{
+    sharded_block_partitioner::config::PartitionerV1Config, v2::config::PartitionerV2Config,
```

Comment:
Is there a good reason for the hierarchies of PartitionerV1Config and PartitionerV2Config to be different ?

Reply:
v1 is actually an alias of `sharded_block_partitioner`, so the hierarchy is the same?

Reply:
I added some clarification comments on top of `PartitionerV1Config`...

Reply:
Then, perhaps call it `sharded_block_partitioner_v2` ?

Reply:
it's tricky... the v2 is also only a multithreaded program rather than really being "sharded"...

Reply:
I'd rather rename the old one to v1, but that unnecessarily makes the current diff bigger...

Reply:
ok

Reply:
added a TODO as well.